XTS/localjunit/unit/src/test/java/com/arjuna/wst/tests/arq/TestCompletionCoordinatorProcessor.java

```diff
@@ -31,7 +31,7 @@
 
 public class TestCompletionCoordinatorProcessor extends CompletionCoordinatorProcessor
 {
-    private Map messageIdMap = new HashMap() ;
+    private Map<String,CompletionCoordinatorDetails> messageIdMap = new HashMap<String,CompletionCoordinatorDetails>() ;
```

Comment:
With the diamond operator one would normally allow the compiler to infer the type here by writing:

private Map<String,CompletionCoordinatorDetails> messageIdMap = new HashMap<>();

and similarly in all the other places in this patch where you are using this syntax.

Reply:
true!